### ISSUE TITLE
Load session for specific directory

### ISSUE THREAD

**Issue by twiddli** [Shatur/neovim-session-manager#80]
**Describe the problem or limitation you are having**

There's a `load_current_dir_session` but not a `load_dir_session` which takes a path to a dir.

**Describe the solution you'd like**

A `load_dir_session` that takes a path to a dir.

**Describe alternatives you've considered**

**Additional context**

If you also add `current_dir_session` that returns `pwd` if a session is active or empty string otherwise, implementing features like #39 or #29 should be a breeze even outside of the plugin.

**Comment by Shatur** [comment]
I agree with the idea, but I would keep only `load_dir_session` and load current dir if the passed path is empty.

**Comment by twiddli** [comment]
Not sure if this warrants a new issue, but would also be nice to have a general `current_sessions` or similar that returns a list of all sessions. The usecase would be to display them on something like a dashboard.

**Comment by pda-gb** [comment]
> Just use this:
> 
> https://github.com/Shatur/neovim-session-manager/blob/d1883f30921193f3cff4537e27514e454e0331e9/lua/session_manager/utils.lua#L114

how to use this in the tmuxinator config ?
`$ nvim -c ':SessionManager load_current_dir_session' ` - open last session
`$ nvim -c ':SessionManager load_session' /path/to/root/project/` -open last session
How to open a custom session via the terminal?

**Comment by Shatur** [comment]
> how to use this in the tmuxinator config ?

The function return list of all session. This is what I was asked about.

>  How to open a custom session via the terminal?

Use `:SessionManager load_session` and it will let you select a session.

**Comment by pda-gb** [comment]
**Upd.:**
`$ nvim -c ':SessionManager [load_current_dir_session/load_sat_session/load_session /path/to/root/project/]'` always does one thing - it does not load the last session. Loads the last opened buffers. I opened a random file not from sessions and through this command was loaded not the specified session, but this file.
I also accidentally made a mistake in init.lua and nvim did not load but crashed with an error. So in the last run there were no open buffers. So I decided to do an experiment: run nvim like this - `$ nvim -c ':SessionManager load_last_session'`.  And since this doesn't actually load the last session, but the last open buffers(which are not there), nvim crashed out with an error.

**Comment by Shatur** [comment]
I don't understand you.

There is no such command `:SessionManager load_session /path/to/root/project/`. `load_session` does not accept any argument. It opens session selection.

> always does one thing - it does not load the last session. Loads the last opened buffers.

Not sure what you meant by that.
See the https://github.com/Shatur/neovim-session-manager#commands for commands and how they works.

**Comment by pda-gb** [comment]
> `load_session` does not accept any argument. It opens session selection

I know. Someone wrote in the comments about this application and I just tried it.  But still all three options :SessionManager do load the last buffers. 
Now running via terminal: nvim -c ':SessionManager load_current_dir_ression' will get an error:


![изображение](https://github.com/Shatur/neovim-session-manager/assets/52175155/8495eaa4-9989-4006-a6f0-6ce448a0497e)

**Comment by pda-gb** [comment]
ок
